integration-tests-support/wiremock/src/main/java/org/apache/camel/quarkus/test/wiremock/WireMockTestResourceLifecycleManager.java

```diff
@@ -217,6 +219,9 @@ private WireMockServer createServer() {
             // Read mapping resources from the classpath in playback mode
             configuration.fileSource(new CamelQuarkusFileSource());
         }
+
```

Comment:
I read it like HTTPs is enabled for all tests. It reminds me that we added the possibility to customize the configuration in specific cases with `customizeWiremockConfiguration()`.

Reply:
@aldettinger thanks for the tips. I'll use that!!